packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -111,13 +119,20 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     if (!this.context.scrollablePane) {
       return true;
     }
-
     const { isStickyTop, isStickyBottom } = this.state;
+    const nonStickyContent = this.nonStickyContent;
+    const nonStickyContentOffsetHeight = nonStickyContent ? nonStickyContent.offsetHeight : undefined;
+    const nonStickyRoot = this.root;
```

Comment:
Shall we just call it 'root'? Lets not create confusion by giving it another name here.